x/rollapp/types/params.go

```diff
@@ -17,10 +18,21 @@ var (
 	// KeyDeployerWhitelist is store's key for DeployerWhitelist Params
 	KeyDeployerWhitelist = []byte("DeployerWhitelist")
 	// KeyDisputePeriodInBlocks is store's key for DisputePeriodInBlocks Params
-	KeyDisputePeriodInBlocks            = []byte("DisputePeriodInBlocks")
+	KeyDisputePeriodInBlocks = []byte("DisputePeriodInBlocks")
+
+	KeyLivenessSlashBlocks   = []byte("LivenessSlashBlocks")
+	KeyLivenessSlashInterval = []byte("LivenessSlashInterval")
+	KeyLivenessJailBlocks    = []byte("LivenessJailBlocks")
```

Comment:
some docstrings would be nice

Reply:
can you suggest a useful docstring?

Reply:
```
// KeyLivenessSlashBlocks is the params store's key that represents the number of blocks until an impending liveness slash.
KeyLivenessSlashBlocks   = []byte("LivenessSlashBlocks")
// KeyLivenessSlashInterval is the params store's key that represents the interval at which slashing will take place, once initiated and until interrupted.
KeyLivenessSlashInterval = []byte("LivenessSlashInterval")
// KeyLivenessJailBlocks is the params store's key that represents the number of blocks until the offending sequencer shall be jailed.
KeyLivenessJailBlocks    = []byte("LivenessJailBlocks")
```

Reply:
hope I got their purpose right

Reply:
right
I don't like this personally because it duplicates the param.proto docstring
bit pointless and will cause drift

Reply:
but then when you want to see what these are about, you need to look all the way in the protos.
Also, what do you mean by drift?

Reply:
Basically if you document the meaning of every value on the key for the value you are duplicating the description
Which is bad because you should just have the source of truth

The only valid docstring for the key is IMO

```
// KeyFoo is the key to lookup Foo in the store
KeyFoo = []byte("foo)"
```

which is useless

also

>you need to look all the way in the protos.

it takes me almost no effort to navigate to protos from param file

Anyways, when there is a source of truth, of course you have to navigate to it